dashboard/templatetags/cards.py

```diff
@@ -22,6 +22,9 @@ def card_diaperchange_last(child):
     """
     instance = models.DiaperChange.objects.filter(
         child=child).order_by('-time').first()
+
+    if not instance and getattr(Settings.objects.first(), 'dashboard_hide_empty'):
```

Comment:
@cdubz, I'm not super familiar with Django, but I imagine there must be a better way to do this?

Reply:
Yeah I don't love that pattern. My recommendation here would be to _not_ exit early in these functions with just the `hidden` prop but to add an `empty` prop to the regular returned object. Don't worry about `Settings.objects` at that point though, just use `user.settings.dashboard_hide_empty` when you check for the `empty` prop in `dashboard/templates/cards/base.html`.